geopandas/geodataframe.py

```diff
@@ -747,6 +747,40 @@ def astype(self, dtype, copy=True, errors="raise", **kwargs):
         # do not return a GeoDataFrame
         return pd.DataFrame(df)
 
+    def to_postgis(
+        self, con, table, if_exists="fail", schema=None, dtype=None, index=False
```

Comment:
Can we keep the same attribute names and order of attributes as pandas has in [to_sql](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.to_sql.html#pandas.DataFrame.to_sql)? For the sake of consistency. E.g. `table` should be `name` and order `self, name, con, schema=None, if_exists='fail', index=True, dtype=None`.

Reply:
Thanks for your review @martinfleis! :) Sure, I'll check that this follows the same style as in Pandas.

Reply:
Now we follow fully the pandas `to_sql` syntax including `chunksize` as well.